data.json

```diff
@@ -368,6 +386,24 @@
     "dateAdded": "2019-10-14T16:16:36.000Z",
     "tags": ["clothing", "hacktoberfest"]
   },
+  {
+    "name": "Saleor",
+    "difficulty": "medium",
+    "description": "Solve any of the issues with the Hacktoberfest label and win a pair of Saleor Socks.",
```

Comment:
This should be an url of an image, it's a web page now.

Reply:
Whoops! My bad. Missed that. I have fixed it now. Please have a look.